force-app/main/default/classes/SOQL.cls

```diff
@@ -259,7 +359,14 @@ public with sharing class SOQL {
      * @return An Account record with related Opportunities.
      */
     public static Account getAccountAndOpportunitiesFromAccountId(Id accountId) {
-        return null; // Replace null with the variable you used to store the result
+        Account a = [
+            SELECT Id, Name, (SELECT Id, Name FROM Opportunities)
+            FROM Account
+            WHERE Id = :accountId
+            LIMIT 1
+        ];
+
```

Comment:
Lets try to do this in one line start with return and use a meaningful variable name of course :)